plugins/ipam/static/README.md

```diff
@@ -38,8 +38,17 @@ static IPAM is very simple IPAM plugin that assigns IPv4 and IPv6 addresses stat
 ## Network configuration reference
 
 * `type` (string, required): "static"
-* `addresses` (array, required): an array of arrays of ip address objects:
+* `addresses` (array, optional): an array of arrays of ip address objects:
 	* `address` (string, required): CIDR notation IP address.
 	* `gateway` (string, optional): IP inside of "subnet" to designate as the gateway.
 * `routes` (string, optional): list of routes add to the container namespace. Each route is a dictionary with "dst" and optional "gw" fields. If "gw" is omitted, value of "gateway" will be used.
 * `dns` (string, optional): the dictionary with "nameservers", "domain" and "search".
+
+## Supported arguments
+The following [CNI_ARGS](https://github.com/containernetworking/cni/blob/master/SPEC.md#parameters) are supported:
```

Comment:
Missing new line before this one. Please keep that in sync with rest of document.

Reply:
Fixed.